src/Harald.WebApi/Application/EventHandlers/CapabilityDeletedEventNotifyDedHandler.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text;
+using System.Threading.Tasks;
+using Harald.Infrastructure.Slack;
+using Harald.WebApi.Domain;
+using Harald.WebApi.Domain.Events;
+
+namespace Harald.WebApi.Application.EventHandlers
+{
+    
+    public class CapabilityDeletedEventNotifyDedHandler : IEventHandler<CapabilityDeletedDomainEvent>
+    {
+        private readonly ISlackFacade _slackFacade;
+
+        public CapabilityDeletedEventNotifyDedHandler(ISlackFacade slackFacade)
+        {
+            _slackFacade = slackFacade;
+        }
+
+        public async Task HandleAsync(CapabilityDeletedDomainEvent domainEvent)
+        {
+            var hardCodedDedChannelId = new ChannelId("GFYE9B99Q");
+            
+            var stringBuilder = new StringBuilder();
```

Comment:
This will actually give us worse performance after the updates to string perf a few years ago. Read more here @ https://michaelscodingspot.com/challenging-the-c-stringbuilder/. Generally you want to reserve StringBuilder for foreach-style concatenation of string arrays.

Reply:
I read the same but forgotten it :/ 
`Code changed to follow recommendations`